### ISSUE TITLE
Plugin does not work at all following the publishing of sail files

### ISSUE THREAD

**Issue by rqpt** [jessarcher/zsh-artisan#28]
## Laravel Version

11.16.0

## PHP Version

8.3.9

## Tldr

Plugin breaks after publishing sail. Fix coming soon.

## Description

The plugin does not work after you have published your sail dockerfiles.

### Input

```bash
artisan make:model
```

### Output

```bash
WARN[0000] The "WWWUSER" variable is not set. Defaulting to a blank string.
WARN[0000] The "WWWGROUP" variable is not set. Defaulting to a blank string.
service "php" is not running container #1
```

## Why is this happening?

When you publish your sail files, the "services.'laravel.test'.build.context" value inside of the docker-compose.yml file changes from **'./vendor/laravel/sail/runtimes/8.3'** to **_'./docker/8.3'_**.

This causes a grep check for 'laravel/sail' to return empty and ultimately returns false inside a conditional check that should be returning true.

### Docker Compose

**Before publishing sail:**
```yaml
services:
    laravel.test:
        build:
            context: ./vendor/laravel/sail/runtimes/8.3
```

**After publishing sail:**
```yaml
services:
    laravel.test:
        build:
            context: ./docker/8.3
```

### Offending code

artisan.plugin.zsh:24-26

```bash
    if [ "`grep "laravel/sail" $docker_compose_config_path | head -n1`" != '' ]; then
      artisan_cmd="$laravel_path/vendor/bin/sail artisan"
    else
```

## Steps To Reproduce

   1. Install laravel.
   2. Install sail.
   3. Publish sail. 
   4. Run any of the plugin artisan commands.

## Fix incoming

I have a fix for this, and will submit a pr soon.

Greetings from a saffer!
